### PR TITLE
Improving kubectl cheat sheet

### PR DESCRIPTION
**Pull request by chrislovecnm**
The one question that I have is that autocomplete is showing different resources names.  I pulled the resource name table from the kubectl-overview.md page.  Most of the resource names appear to now be singular.  Anyways good stab at an update of the cheat sheet.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1486)

### CODE REVIEW

File: docs/user-guide/kubectl-cheatsheet.md [hunk 1]
```diff
@@ -5,22 +5,54 @@ assignees:
 - krousey
 
 ---
-An assortment of compact kubectl examples
+A cheat sheet for `kubectl`.
```
Comment (devin-donnelly) [inline docs/user-guide/kubectl-cheatsheet.md]: Why? The title is already "Kubectl cheat sheet", which is not self-explanatory for non-English speakers.

Suggest changing to "This document contains a quick reference for common tasks you can accomplish with `kubectl`."

Comment (chrislovecnm) [inline docs/user-guide/kubectl-cheatsheet.md]: How about we remove the line completely.  To me a cheat sheet needs to be terse.

Comment (devin-donnelly) [inline docs/user-guide/kubectl-cheatsheet.md]: That's fine by me.

**Comment by chrislovecnm** [comment]
@devin-donnelly updated.

**Comment by chrislovecnm** [comment]
@devin-donnelly I have made the changes ... when do you have time to review?

**Comment by chrislovecnm** [comment]
@bgrant0607 would like to get this in ... can someone tap @devin-donnelly on the shoulder for me?

**Comment by bgrant0607** [comment]
@chrislovecnm Please ping @kubernetes/docs instead.

**Comment by chrislovecnm** [comment]
Thanks ;) will do. @kubernetes/docs can I get an assist on this.

**Pull request merged**